tests/rest/test-rest-api.php

```diff
@@ -107,7 +107,7 @@ public function test_rest_api_returns_blocks_for_post() {
 					'mediaId'           => 6,
 					'mediaLink'         => 'https://gutenberg-block-data-api-test.go-vip.net/?attachment_id=6',
 					'mediaType'         => 'image',
-					'align'             => 'wide',
+					'align'             => 'none',
```

Comment:
From my local testing, it looks like the default value for this has changed so this keeps it in line with that. Wonder if it's a 6.3 change?

Reply:
Good catch! Looks like this was [changed 6 months ago in Gutenberg](https://github.com/WordPress/gutenberg/commit/ced52228a39ba68fea86c1e3e8d36aa7947685db#diff-4d936a4cb04a450622ce03fd585affd8c3bccb3982d49faae3201ae71dbbf940L13), which was later [merged into WordPress 6.3](https://github.com/WordPress/WordPress/commit/00401e16b30c2929e8adb5e07521b3ace4dbf826) like you said.